docs/index.html

```diff
@@ -46,6 +46,11 @@ <h2 class="title">A 1x Engineer...</h2>
         <li>Supports code in production, even if they did not write it.</li>
         <li>Can feel like an impostor at times.</li>
         <li>Makes sure discussions are always productive and everyone gets to have their say before the team makes a decision. Only disregards opinions by providing arguments.</li>
+        <li>Create community and share your knowledge</li>
+        <li>Never stop learning</li>
+        <li>Always leave your comfort zone</li>
```

Comment:
On this one, I'd say "be willing to" rather than "always" since it's exceptionally challenging for some people to leave their comfort zones given certain mental health conditions.

```suggestion
        <li>Be willing to leave your comfort zone.</li>
```